sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/active/directory/implementation/autoconfigure/b2c/AADB2CProperties.java

```diff
@@ -157,6 +156,35 @@ private void validateCommonProperties() {
         }
     }
 
+    /**
+     * Validate URL properties configuration.
+     */
+    private void validateURLProperties() {
+        if (!isValidURL(logoutSuccessUrl)) {
+            throw new AADB2CConfigurationException("logout success should be valid URL.");
+        }
+        if (!isValidURL(baseUri)) {
+            throw new AADB2CConfigurationException("baseUri should be valid URL.");
+        }
+    }
+
+    /**
+     * Used to validate uri, the uri is allowed to be empty.
+     * @param uri
+     * @return whether is uri is valid or not.
+     */
+    private boolean isValidURL(String uri) {
```

Comment:
this should be a more common util, thus should be located in a util class in core , this properties class is not the proper place